osbenchmark/worker_coordinator/worker_coordinator.py

```diff
@@ -220,22 +223,27 @@ def __init__(self):
         self.cluster_details = None
 
     def receiveMsg_PoisonMessage(self, poisonmsg, sender):
-        self.logger.error("Main worker_coordinator received a fatal indication from load generator (%s). Shutting down.", poisonmsg.details)
+        self.logger.error(
+            "Main worker_coordinator received a fatal indication from load generator (%s). Shutting down.", poisonmsg.details)
```

Comment:
A lot of these changes in worker_coordinator.py are new lines in or reformatting existing lines. Are these intentional or was this done with an automated tool? We should minimize changes that aren't related to the proposed feature